src/survey-public/components/feedback/FeedbackSecondPage.tsx

```diff
@@ -96,21 +109,37 @@ export const FeedbackSecondPage: React.FC<Props> = ({
           }
         })}
       </div>
+      {isButtonTouched && (
+        <SurveyFormErrorsBox
+          customErrors={
+            watchIsMap === "1" && !watchIsLocationValue
```

Comment:
Die Bedingung kommt ja mehrfach und sie ist nicht sehr sprechend. Gerne oben in eine gemeinsame Variable ziehen und Erklärungscomment drüber.